openapi/data_object_service.swagger.yaml

```diff
@@ -560,6 +560,13 @@ definitions:
     description: |-
             OPTIONAL
             A set of key-value pairs that represent metadata provided by the uploader.
+  AuthorizationMetadata:
+    type: object
+    additionalProperties: true
+    description: |-
+            OPTIONAL
+            A set of key-value pairs that represent sufficient metadata to be granted 
```

Comment:
Don't we need some structure here?  Like if I was building a generic DOS client how would I know what these fields would be ahead of time?  And if I wanted to support a particular service but everyone describes that service with different terms how will that work?